dts/bindings/led/awinic,aw20216s.yaml

```diff
@@ -0,0 +1,27 @@
+description: Awinic AW20216S LED Driver
+
+compatible: "awinic,aw20216s"
+
+include: ["spi-device.yaml", "led-controller.yaml"]
+
+on-bus: spi
+
+properties:
+  en-gpios:
+    type: phandle-array
+    required: false
+    description: Enable pin
+  sync-gpios:
+    type: phandle-array
+    required: false
+    description: Sync pin
+  current-limit:
+    type: int
+    default: 150
```

Comment:
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Also, is this really expressed in mA or isn't it rather a unitless number between 0 and 0xFF. Couldn't tell for sure with a quick look at the datasheet.

Reply:
I think this may be more complicated since it is a fraction of the current limit set by an external resistor. Hmmmm.

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/36466317/d7116a51-4063-4d6c-b24f-185bf65c9aa0)

Reply:
@gmarull may be able to suggest something here